ttml.go

```diff
@@ -394,7 +405,7 @@ func ReadFromTTML(i io.Reader) (o *Subtitles, err error) {
 			// New line decoded as a line break. This can happen if there's a "br" tag within the text since
 			// since the go xml unmarshaler will unmarshal a "br" tag as a line break if the field has the
 			// chardata xml tag.
-			for idx, li := range strings.Split(tt.Text, "\n") {
+			for idx, li := range strings.Split(tt.Text, "\\n") {
```

Comment:
Can you provide an example `ttml` that was having issue with that bit of code?

Reply:
I used testdata>example-in.ttml, added '\n' n tested, it was not working.

Reply:
What do you mean `it was not working`? What did you expect to see?

Reply:
The code is supposed to spilt the string on '\n' but it was not splitting so I added this extra '\' and now it's working.

Reply:
Mmmm I'm really not sure `.ttml` text should be split on string `\n` 🤔 I think it should only be split on line returns.

Can you share some official docs or RFC that provide the same information as you?

Reply:
I also think the same, but the code was written to split on new line characters, and during my testing, this code to split was not working, so I added this extra slash.
Anyway, removing it.